cmd/grant.go

```diff
@@ -15,19 +15,25 @@
 package cmd
 
 import (
+	"errors"
+
 	"github.com/magneticio/forklift/logging"
 	"github.com/magneticio/vampkubistcli/client"
 	"github.com/spf13/cobra"
 )
 
 var Role string
+var Permission string
+var Kind string
 
 // grantCmd represents the grant command
 var grantCmd = &cobra.Command{
 	Use:   "grant",
 	Short: "Grant a role or permission to a user for an object",
 	Long: AddAppName(`Usage:
-$AppName grant --user user1 --role admin -p default`),
+$AppName grant --user user1 --role admin -p default
+$AppName grant --user user1 --permission rwda -p project -c cluster -r virtualcluster -a application --kind deployment --name deploymentname
```

Comment:
Can you add an explanation what rwda stands for? "editAccess" is also is not obvious.